No configuration file found.
No pyproject.toml file found.
stubPath /mnt/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/typings is not a valid directory.
Assuming Python platform Linux
Searching for source files
Found 1 source file
/home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:25:8 - error: Import "apache_beam" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:26:8 - error: Import "pyarrow" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:28:8 - error: Import "tensorflow_data_validation" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:29:8 - error: Import "tensorflow_transform" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:30:6 - error: Import "tensorflow_transform" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:31:8 - error: Import "tensorflow_transform.beam" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:32:6 - error: Import "tensorflow_transform.beam" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:33:6 - error: Import "tensorflow_transform.beam" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:34:6 - error: Import "tensorflow_transform.saved" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:35:6 - error: Import "tensorflow_transform.tf_metadata" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:36:6 - error: Import "tensorflow_transform.tf_metadata" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:37:6 - error: Import "tensorflow_transform.tf_metadata" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:38:6 - error: Import "tensorflow_transform.tf_metadata" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:39:6 - error: Import "tfx" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:40:6 - error: Import "tfx.components.base" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:41:6 - error: Import "tfx.components.transform" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:42:6 - error: Import "tfx.components.transform" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:43:6 - error: Import "tfx.components.util" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:44:6 - error: Import "tfx.proto" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:45:6 - error: Import "tfx.types" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:46:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:47:6 - error: Import "tfx.utils" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:48:8 - error: Import "tfx_bsl" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:49:6 - error: Import "tfx_bsl.tfxio" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:50:6 - error: Import "tfx_bsl.tfxio" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:51:6 - error: Import "tfx_bsl.tfxio" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:54:6 - error: Import "tensorflow_metadata.proto.v0" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:55:6 - error: Import "tensorflow_metadata.proto.v0" could not be resolved (reportMissingImports)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:546:27 - error: Argument missing for parameter "stats_output_path" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:983:45 - error: Argument missing for parameter "analyze_paths_count" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1057:49 - error: Cannot access member "key" for type "tuple[Unknown, ...]"
    Member "key" is unknown (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1112:61 - error: Argument missing for parameter "schema" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1151:22 - error: Argument missing for parameter "pcoll" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1164:14 - error: Argument missing for parameter "schema" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1176:57 - error: Argument missing for parameter "schema" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1185:53 - error: Argument missing for parameter "transformed_example_path" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1290:28 - error: Expression of type "list[None]" cannot be assigned to declared type "Sequence[Text] | None"
    Type "list[None]" cannot be assigned to type "Sequence[Text] | None"
      TypeVar "_T_co@Sequence" is covariant
        Type "None" cannot be assigned to type "Text"
      Type cannot be assigned to type "None" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1294:34 - error: Expression of type "list[None]" cannot be assigned to declared type "Sequence[Text] | None"
    Type "list[None]" cannot be assigned to type "Sequence[Text] | None"
      TypeVar "_T_co@Sequence" is covariant
        Type "None" cannot be assigned to type "Text"
      Type cannot be assigned to type "None" (reportGeneralTypeIssues)
  /home/apr/data/2020-11-API-misuse/Repair_API_Misuse/api-misuse-repair/2_parse_dl_repos/repo_download_v2/misuse_repos/tensorflow_tfx/a57ac7965bf6d74ed9e9660b086e3b9df3b24533/a/tfx/components/transform/executor.py:1298:27 - error: No overloads for "__new__" match the provided arguments
    Argument types: (Sequence[Text | int], Sequence[Text | int], Sequence[Text] | None, Sequence[Text] | None) (reportGeneralTypeIssues)
39 errors, 0 warnings, 0 infos 
Completed in 1.813sec

time cost(in seconds): 1.99369478225708
